_inc/client/plans/plan-body.jsx

```diff
@@ -132,13 +161,13 @@ const PlanBody = React.createClass( {
 								<p>{ __( 'Earn income by allowing Jetpack to display high quality ads (powered by WordAds).' ) }</p>
 								{
 									this.props.isModuleActivated( 'wordads' ) ? (
-										<Button href={ 'https://wordpress.com/ads/earnings/' + this.props.siteRawUrl } className="is-primary">
+										<Button onClick={ () => this.trackPlansClick( 'view_earnings' ) } href={ 'https://wordpress.com/ads/earnings/' + this.props.siteRawUrl } className="is-primary">
```

Comment:
I know, it's a little less performant but honestly quite ridiculous to not be able to reuse a function like this.  I make a decision in this case to go for maintainability/readability